### DISCUSSION TITLE
No module named 'onnxruntime' but its installed? help

### DISCUSSION THREAD

**Discussion by iwannalikelinux** [Stability-AI/StableSwarmUI#466] [General]
I am very new Linux mint; so far have been able to install StableSwarmUI and generate few pics using cuda on 3060ti

I realized to work with 3d models I should using the workflows tab – I am amazed how intuitive as software StableSwarmUI and ComfyUI are :)

I installed the package manger to workflow

this is where I encounter problems when i run workflow and node - RemBGSession
i get following error:

RemBGSession+
No module named 'onnxruntime'

My first thought was i need to get onnxruntime

pip install torch==2.0.1+cu118 torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu118
pip install --upgrade onnxruntime-gpu
pip install rembg

I run again same error message?

I have tried ask AI i got this:

pip uninstall torch torchvision torchaudio onnxruntime-gpu -y

Reinstall PyTorch with CUDA support (make sure to match versions)
pip install torch==2.1.1+cu118 torchvision==0.16.1+cu118 torchaudio==2.1.1+cu118 -f https://download.pytorch.org/whl/torch_stable.html

Then reinstall ONNX Runtime again.
pip install onnxruntime-gpu==1.14.0

finally i tried:
pip install --upgrade onnxruntime-gpu

same issue

RemBGSession+
No module named 'onnxruntime'

I really want try different workflows on comfy I have seen…

I cant resolve this issue after 5+ hours I need help, I don’t get python or how I have seemingly installed what was requested yet its not being seen?

Any help would be really appreciated, please remember I was total newbie (I am trying to learn this)

System Information
ComfyUI Version: v0.3.7-33-g6d1a3f7
Arguments: main.py --port 7821 --extra-model-paths-config //// StableSwarmUI /StableSwarmUI/Data/comfy-auto-model.yaml --preview-method latent2rgb
OS: posix
Python Version: 3.12.3 (main, Sep 11 2024, 14:17:37) [GCC 13.2.0]
Embedded Python: false
PyTorch Version: 2.5.1+cu121
Devices
Name: cuda:0 NVIDIA GeForce RTX 3060 Ti : cudaMallocAsync
Type: cuda
VRAM Total: 8350400512
VRAM Free: 6787563520
Torch VRAM Total: 0
Torch VRAM Free: 0
/StableSwarmUI/dlbackend/ComfyUI/execution.py", line 328, in execute
22:19:45.445 [Warning] ComfyUI-0 on port 7821 stderr: output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
22:19:45.445 [Warning] ComfyUI-0 on port 7821 stderr: ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

I have also tried : 
https://onnxruntime.ai/docs/install/#install-onnx-runtime-gpu-cuda-11x

**Comment by mcmonkey4eva** [comment]
Did you miss the readme migration notice? https://github.com/Stability-AI/StableSwarmUI?tab=readme-ov-file#migration-notice
https://github.com/mcmonkeyprojects/SwarmUI/discussions/2

**Comment by iwannalikelinux** [comment]
thank you that seems to have helped!
My workflow is running but im now getting this error

TripoSRSampler
Cannot handle this data type: (1, 1, 5), |u1

but at least im past the first issue :)

**Comment by mcmonkey4eva** [comment]
any support topics related to SwarmUI, post on the correct modern repo and/or the Discord, not this legacy Stability repo
